package gift.member.exception;

import gift.common.exception.CustomException;
import org.springframework.http.HttpStatus;

public class EmailAlreadyExistsException extends CustomException {
    public EmailAlreadyExistsException(String email) {
        super("이미 사용 중인 이메일입니다: " + email, HttpStatus.CONFLICT);
    }
}